sdk/keyvault/keyvault-admin/test/utils/common.ts

```diff
@@ -56,3 +60,33 @@ export function getSasToken() {
 
   return { blobStorageUri, blobSasToken };
 }
+
+/**
+ * The known API versions that we support.
+ */
+export const serviceVersions = ["7.0", "7.1", "7.2", "7.3-preview"] as const;
```

Comment:
I wonder if 7.0, 7.1 would even work here given than this package started with 7.2 support - anyway I think it's safe to limit our test matrix to 7.2 and 7.3-preview for kv-admin

Reply:
ok ok thank you! 🙏

Reply:
Done! only 7.2 though! I’m not sure how to add 7.3-preview in the platform-matrix

Reply:
yeah, sorry - only 7.2 😄 7.3-preview is the current and default version so it's already covered. We only need additional entries for _previous_ versions so only 7.2 makes sense 👍